conans/client/generators/cmake_common.py

```diff
@@ -735,13 +735,13 @@ class CMakeCommonMacros:
                         set(CONAN_FRAMEWORKS ${CONAN_FRAMEWORKS_DEBUG} ${CONAN_FRAMEWORKS})
                         set(CONAN_FRAMEWORK_DIRS ${CONAN_FRAMEWORK_DIRS_DEBUG} ${CONAN_FRAMEWORK_DIRS})
                     elseif(${CMAKE_BUILD_TYPE} MATCHES "Release")
-                        set(CONAN_FRAMEWORKS ${CONAN_FRAMEWORKS_RELEASE} ${CONAN_BUILD_MODULES_PATHS})
+                        set(CONAN_FRAMEWORKS ${CONAN_FRAMEWORKS_RELEASE} ${CONAN_FRAMEWORKS})
```

Comment:
I thought that was already fixed in another PR.
Maybe this diff is not updated? A rebase of this PR from develop might fix this.

Reply:
Yes, done! GitHub didn't update it